src/main/java/org/mule/extension/aggregator/internal/privileged/executor/AbstractAggregatorExecutor.java

```diff
@@ -213,6 +222,30 @@ public void start() throws MuleException {
     }
   }
 
+  private void executeDeferActions() {
+    quorumManager = new Thread(() -> {
+      boolean startCompleted = false;
+      while (!startCompleted) {
+        try {
+          Thread.sleep(200);
+          upgradeAggregatedContentIfNeeded();
+          setRegisteredAsyncAggregationsAsNotScheduled();
+          startCompleted = true;
+        } catch (Exception e) {
+          if (e.getClass().getName().contains(QUORUM_EXCEPTION)) {
```

Comment:
Why are you handling the QuorumException like this? Why not catch it by the exception class directly?

Reply:
We have to react to all kinds of exceptions, in a particular case, we want to log a specific message, I think bringing the whole dependency just to get access to an exception class might be unnecessary.